docs/src/tutorial-nondeterministic-variables.md

```diff
@@ -1,95 +1,105 @@
 # Nondeterministic variables
 
-Kani is able to reason about programs and their execution paths by allowing users to assign nondeterministic (i.e., symbolic) values to  certain variables.
-Since Kani is a bit-level model checker, this means that Kani considers that an unconstrained nondeterministic value represents all the possible bit-value combinations assigned to the variable's memory contents.
+Kani is able to reason about programs and their execution paths by allowing users to assign nondeterministic (also called symbolic) values to certain variables using `kani::any()`.
+Kani is a "bit-precise" model checker, which means that Kani considers all the possible bit-value combinations that could be assigned to the variable's memory contents.
 
 As a Rust developer, this sounds a lot like the `mem::transmute` operation, which is highly `unsafe`.
 And that's correct.
+But `kani::any()` is only implemented for a few types: those where we are able to safely enforce the type's invariants.
```

Comment:
I think this will beg the question: "What should I do if I need `kani::any()` for one of the unimplemented types?" So perhaps add a reference to the section that discusses that, e.g:

(See Section X for a discussion on how to assign nondeterministic values to types that `kani::any()` is not implemented for)